packages/eslint-plugin/tests/areOptionsValid.test.ts

```diff
@@ -17,16 +17,16 @@ const exampleRule = createRule<['value-a' | 'value-b'], never>({
   name: 'my-example-rule',
 });
 
-test('returns true for valid options', () => {
+it('returns true for valid options', () => {
   expect(areOptionsValid(exampleRule, ['value-a'])).toBe(true);
 });
 
 describe('returns false for invalid options', () => {
-  test('bad enum value', () => {
+  it('bad enum value', () => {
```

Comment:
🤔 I actually like `test()` here over `it()`. The test name doesn't grammatically read out as a full sentence. I've personally kind of gravitated towards:

* `it()`: for most tests, if there's at all a way to phrase it as _"it X when Y"_ or something like that
* `test()`: as a fallback if there's no logic, just a descriptor - like _"test X"_

Reply:
I don't really care which one we use as long as it is one, and not two 

If you can get the others to agree, I'm happy to update

Reply:
Oh don't get me wrong, I think `it()` is the right choice almost all of the time. Just my _personal nitpicky preference_ is to use `test()` in _specific situations_. 😄

Reply:
To be honest, my preference is actually suite/test/assert. But the most consistent thing we can do right now is `it` it seems

As a middle ground, id prefer to later move from the now-consistent `it` to `test ` if anything